### PR TITLE
Homework1

### PR DESCRIPTION
**Pull request by ZarechOleksii**

### CODE REVIEW

File: C#/homework1/homework1/Collection.cs [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+namespace homework1
+{
+    class Collection
+    {
+        private List<Event> coll = new List<Event>();
+        private string filePath;
+        public Collection(string fileName)
+        {
+            JArray obj;
+            filePath = "D:\\School\\C# LNY\\Navch Prakt\\homework1\\homework1\\" + fileName;
```
Comment (tetianastoyko) [inline C#/homework1/homework1/Collection.cs]: remove hardcode

File: C#/homework1/homework1/Collection.cs [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+                obj_counter++;
+            }
+        }
+
+        public void Add(Event to_add)
+        {
+            coll.Add(to_add);
+        }
+
+        public bool Present(int giv_id)
```
Comment (tetianastoyko) [inline C#/homework1/homework1/Collection.cs]: Present -> isPresent (as bool result is returned)

File: C#/homework1/homework1/RestaurantNames.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace homework1
+{
+    public enum RestaurantNames
```
Comment (tetianastoyko) [inline C#/homework1/homework1/RestaurantNames.cs:9]: nice

File: C#/homework1/homework1/main.cs [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+                            }
+                            break;
+                        case 'D':
+                            Console.Write(allEvents.PrintAll());
+                            Console.WriteLine("Select the Event to delete:");
+                            string to_delete = Console.ReadLine();
+                            Console.WriteLine(allEvents.DeleteOne(to_delete));
+                            allEvents.Overwrite(); break;
+                        case 'E':
+                            Console.Write(allEvents.PrintAll());
```
Comment (tetianastoyko) [inline C#/homework1/homework1/main.cs]: Case "E" and "A" almost copy-paste. Fix it.

**Review by tetianastoyko — CHANGES_REQUESTED** [review]

File: C#/homework1/homework1/Collection.cs (hunk 1 above)
Comment (ZarechOleksii) [reply C#/homework1/homework1/Collection.cs]: done

File: C#/homework1/homework1/Collection.cs (hunk 2 above)
Comment (ZarechOleksii) [reply C#/homework1/homework1/Collection.cs]: done

File: C#/homework1/homework1/RestaurantNames.cs (hunk 3 above)
Comment (ZarechOleksii) [reply C#/homework1/homework1/RestaurantNames.cs:9]: nice

File: C#/homework1/homework1/main.cs (hunk 4 above)
Comment (ZarechOleksii) [reply C#/homework1/homework1/main.cs]: done

**Pull request merged**